tests/testServerWithStorage.js

```diff
@@ -78,17 +78,119 @@ var testStoryModel = {
     "translationOf": null
 };
 
+var blankStory = {
+    "title": "",
+    "content": [],
+    "contentString": "",
+    "author": "",
+    "tags": [
+        ""
+    ],
+    "keywordString": "",
+    "summary": "",
+    "thumbnailUrl": "",
+    "thumbnailAltText": "",
+    "contentTypes": [],
+    "languageFromSelect": "",
+    "languageFromInput": ""
+};
+
+var blankStoryWithEmptyMediaBlocks = {
+    "title": "",
+    "content": [
+        {
+            "id": null,
+            "language": null,
+            "heading": null,
+            "blockType": "text",
+            "text": null,
+            "simplifiedText": null,
+            "contentString": "",
+            "languageFromSelect": "",
+            "languageFromInput": ""
+        },
+        {
+            "id": null,
+            "language": null,
+            "heading": null,
+            "blockType": "image",
+            "imageUrl": null,
+            "alternativeText": null,
+            "description": null,
+            "contentString": "",
+            "languageFromSelect": "",
+            "languageFromInput": "",
+            "fileDetails": null
+        },
+        {
+            "id": null,
+            "language": null,
+            "heading": null,
+            "mediaUrl": null,
+            "alternativeText": null,
+            "description": null,
+            "transcript": null,
+            "hasTranscript": true,
+            "blockType": "audio",
+            "contentString": "",
+            "languageFromSelect": "",
+            "languageFromInput": "",
+            "fileDetails": null
+        },
+        {
+            "id": null,
+            "language": null,
+            "heading": null,
+            "mediaUrl": null,
+            "alternativeText": null,
+            "description": null,
+            "transcript": null,
+            "hasTranscript": true,
+            "blockType": "video",
+            "contentString": "",
+            "languageFromSelect": "",
+            "languageFromInput": "",
+            "fileDetails": null
+        }
+    ],
+    "contentString": "",
+    "author": "",
+    "tags": [
+        ""
+    ],
+    "keywordString": "",
+    "summary": "",
+    "thumbnailUrl": "",
+    "thumbnailAltText": "",
+    "contentTypes": [],
+    "languageFromSelect": "",
+    "languageFromInput": ""
+};
+
+// TODO: Generalize story testing so that components (such as request
```

Comment:
Perhaps this TODO is better moved to a Jira in order to track it more easily. It could be added to the testing board in that case: https://issues.fluidproject.org/secure/RapidBoard.jspa?rapidView=5&view=planning.nodetail